src/components/itemhelper.js

```diff
@@ -272,7 +272,13 @@ define(['apphost', 'globalize'], function (appHost, globalize) {
 
         canRate: function (item) {
 
-            if (item.Type === 'Program' || item.Type === 'Timer' || item.Type === 'SeriesTimer' || item.Type === 'CollectionFolder' || item.Type === 'UserView' || item.Type === 'Channel') {
+            if (item.Type === 'Program' ||
+                item.Type === 'Timer' ||
+                item.Type === 'SeriesTimer' ||
+                item.Type === 'CollectionFolder' ||
+                item.Type === 'UserView' ||
+                item.Type === 'Channel' ||
+                !item.UserData) {
```

Comment:
Can you put the operators at the beginning of the line rather than the end?

Reply:
https://github.com/jellyfin/jellyfin-web/pull/548/commits/b77143797c6b846f0c6074485b2acc899d4eb746